changelogs/unreleased/fix_release_command.yml

```
@@ -0,0 +1,6 @@
+description: Ensure the `inmanta module release` command never decreases the version number
+change-type: patch
+destination-branches: [master, iso6, iso5]
```

I don't think the command exists on iso5
```suggestion
destination-branches: [master, iso6]
```